plugins/DuskVerb/tools/tuner/outdiff_kurtosis_sweep.py

```
@@ -48,29 +48,35 @@ def obj(trial):
-        nb = glob.glob(f"{d}/*_noiseburst.wav")
-        if r.returncode != 0 or not nb: return 1e3
-        ov = tail_kurt(nb[0], 2000, 14000)
-        if ov is None: return 1e3
-        pen = abs(ov - a_overall)                          # overall toward anchor
-        for b in SUB[1:]:                                  # 4-6/6-9/9-14k toward ~5
-            s = tail_kurt(nb[0], *b)
-            if s is None: return 1e3
-            pen += 0.6*max(0.0, s - max(a_sub[b], 5.0))
-        pen += 0.5*max(0.0, a_hf - hf_level(nb[0]) - 1.0)  # don't dull >1 dB below anchor HF
-        return pen
+        d = f"/tmp/odk_{os.getpid()}_{trial.number}"; shutil.rmtree(d, ignore_errors=True); os.makedirs(d)
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/odk_"

(S108)